packages/@react-spectrum/s2/stories/TagGroup.stories.tsx

```diff
@@ -41,13 +41,34 @@ export let Example = {
     if (args.onRemove) {
       args.onRemove = action('remove');
     }
+    // TODO how to fix hidden/inert dom containing an element with a duplicate id? "foo" will be rendered into real dom twice
     return (
-      <TagGroup {...args}>
-        <Tag>Chocolate</Tag>
-        <Tag>Mint</Tag>
-        <Tag>Strawberry</Tag>
-        <Tag>Vanilla</Tag>
-      </TagGroup>
+      <div style={{width: 320, resize: 'horizontal', overflow: 'hidden', padding: 4}}>
+        <TagGroup {...args}>
```

Comment:
Can we add a story with dynamic collections to make sure that works too? That ts-ignore would have caught that issue